FFMediaToolkit/Encoding/MediaOutput.cs

```diff
@@ -33,14 +35,14 @@ internal MediaOutput(OutputContainer mediaContainer, VideoEncoderSettings videoS
         ~MediaOutput() => Dispose();
 
         /// <summary>
-        /// Gets the video stream in the media file.
+        /// Gets the video streams in the media file.
         /// </summary>
-        public VideoOutputStream Video { get; }
+        public VideoOutputStream[] Video { get; }
```

Comment:
I think we could keep the singular property and have new one VideoStreams with all of them, similar for Audio for consistency

Reply:
Great idea! I'll make sure to fix this.